scripts/utils.ts

```diff
@@ -84,7 +86,20 @@ export async function setupApis(
   // is necessary for generating the SDK (as part of the user agent header).
   // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
   apis.metadata.sdkVersion = await readJsonSync(PACKAGE_JSON).version;
-  await apis.init();
+
+  // Editing the name of the Shopper Context API Model so our name is used for generating class name.
+  // This is hard-coded for now but in the future if we handle the case where type name and title clash better, this can be removed.
+  const shopperContextApi = apis.children.find(
```

Comment:
can we extract this code into a specific function or method (updateShopperContextName). That way it is clean and easy to understand when we want to remove the hack